Apps/frontend/src/const.js

```diff
@@ -34,3 +34,4 @@ export const MIN_SWEEP = 0.5; // <= 1
 export const MAX_SWEEP = 2.0; // >= 1
 export const MIN_AMPLITUDE = 0.0;
 export const MAX_AMPLITUDE = NUM_INTERVALS_HORIZONTAL / 2;
+export const WAVE_CURSOR_SIZE = 50;
```

Comment:
Can we rename to `WAVE_CURSOR_WIDTH`, would be more in line with the rest of the code :)